be/src/fs/fs_starlet.cpp

```diff
@@ -432,45 +442,24 @@ class StarletFileSystem : public FileSystem {
     }
 
 private:
+    void get_s3_store_conf(const staros::starlet::ShardInfo& shard_info, Configuration* conf) {
+        const auto&& s3_store_info = shard_info.get_s3_store_info();
```

Comment:
`const auto&` or `auto&` or `auto&&` but not `const auto&&`: https://stackoverflow.com/questions/13230480/what-does-auto-tell-us

Reply:
done